Framework/Algorithms/src/GetDetOffsetsMultiPeaks.cpp

```diff
@@ -1177,15 +1178,15 @@ void GetDetOffsetsMultiPeaks::addInfoToReportWS(
   m_infoTableWS->cell<double>(wi, 7) = offsetresult.highestpeakdev;
 
   // Peak width delta(d)/d
-  m_resolutionWS->dataX(wi)[0] = static_cast<double>(wi);
+  m_resolutionWS->mutableX(wi)[0] = static_cast<double>(wi);
```

Comment:
I guess this would be a more complicated change to be made (which you do not have to attempt right now), but you may notice that the non const access to the HistogramX/Y/E types occur in a loop. If you check how `addInfoToReportWS` is used.